tests/test_captions.py

```diff
@@ -14,27 +15,59 @@
 
 @pytest.mark.usefixtures('mock_client', 'make_mock_response')
 class TestCaptionEndpoint():
-    def test_get_captions(self, mock_client, make_mock_response):
+    def test_get_captions_default_content_type(self, mock_client, make_mock_response):
         data = 'Test'
+        expected_content_type = CaptionType.SRT.value
         response = make_mock_response(url=URL, text=data)
         mock_client.session.request.return_value = response
 
         res = mock_client.get_captions(JOB_ID)
+        assert res == data
+        mock_client.session.request.assert_called_once_with(
+            "GET",
+            URL,
+            headers={'Accept': expected_content_type}
+        )
 
+    @pytest.mark.parametrize('content_type', [CaptionType.SRT, CaptionType.VTT])
+    def test_get_captions_with_content_type(self, content_type, mock_client, make_mock_response):
+        data = 'Test'
+        expected_content_type = content_type.value
+        response = make_mock_response(url=URL, text=data)
+        mock_client.session.request.return_value = response
+
+        res = mock_client.get_captions(JOB_ID, content_type)
         assert res == data
         mock_client.session.request.assert_called_once_with(
             "GET",
             URL,
-            headers={'Accept': 'application/x-subrip'}
+            headers={'Accept': expected_content_type}
+        )
+
+    def test_get_captions_with_speaker_channel(self, mock_client, make_mock_response):
```

Comment:
Does this test assert anything different from `test_get_captions_default_content_type`?

Reply:
It asserts that the url is correct when we add the speaker channel specification